dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java

```diff
@@ -56,14 +56,17 @@
 import javafx.scene.effect.InnerShadow;
 import javafx.scene.input.*;
 import javafx.scene.layout.StackPane;
+import javafx.util.Pair;
```

Comment:
This is new to me, and useful.  Thanks for showing me!

Reply:
I really like Pair